contracts/vaults/STETHVault.sol

```diff
@@ -133,6 +140,8 @@ contract STETHVault is BaseVault {
         if (supply != 0) {
             if (totalAssets() >= lastRoundAssets) {
                 roundAccruedInterest = totalAssets() - lastRoundAssets;
+                // Notice that `investmentAmount` will be truncated to 0 if
```

Comment:
This scenario could be tested with a unit test

Reply:
Also, now that `INVESTOR_RATIO` is a parameter, other edge case values could also be tested in addition to `DENOMINATOR + 1`

- `0`
- `DENOMINATOR`